src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace System.Net.Http
 {
-    [Serializable]
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
```

Comment:
Shouldn't the SuppressMessage line be removed also since it is dealing with Serialization?